src/components/Map/USACountyMap.js

```diff
@@ -91,45 +88,41 @@ const USACountyMap = React.memo(
               <Geographies geography={geoStates}>
                 {({ geographies }) =>
                   geographies
-                    .filter(geo => isStateFips(geo.id))
+                    .filter(geo => stateFipsCodes.includes(geo.id))
                     .map(geo => {
-                      const { name } = geo.properties;
                       const fipsCode = geo.id;
-                      const stateCode = getStateCode(name);
-                      const stateUrl = `/${getCanonicalUrl(fipsCode)}`;
-                      const locationName = getLocationNameForFips(fipsCode);
-
+                      const state = regions.findByFipsCode(fipsCode);
                       // Using a custom SVG to place the northern mariana islands to increase
                       // accessibility due to the small size.
-                      if (stateCode === 'MP') {
+                      if (state.stateCode === 'MP') {
                         return (
                           <Link
-                            key={stateCode}
-                            to={stateUrl}
-                            aria-label={locationName}
+                            key={state.stateCode}
+                            to={state.relativeUrl}
```

Comment:
this needs to be `/${state.relativeUrl}` I believe

Reply:
I'm going to update this in the class on a separate PR